src/proxy/inbound.rs

```diff
@@ -656,7 +657,8 @@ mod test {
                 namespace: "gatewayns".to_string(),
                 hostname: "gateway".to_string(),
             }),
-            port: 15006,
+            hbone_mtls_port: 15006,
+            hbone_single_tls_port: Some(15006),
```

Comment:
Maybe dumb q - why are these the same here?

Reply:
Lol good catch; typo